src/test/java/Elements/RoomDisplayed_Elements.java

```diff
@@ -0,0 +1,21 @@
+package Elements;
```

Comment:
remove this package, use the package of the page for all elements. each class must be the name of that page. e.g HomePage, RoomsPage

Reply:
Done